pkg/diagnose/ports/ports.go

```diff
@@ -0,0 +1,114 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package ports provides a diagnose suite for the ports used in the agent configuration
+package ports
+
+import (
+	"fmt"
+	"path"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/aggregator/sender"
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/diagnose/diagnosis"
+	"github.com/DataDog/datadog-agent/pkg/util/port"
+)
+
+// func init() {
+// 	diagnosis.Register("port-conflict", diagnosePortSuite)
+// }
+
+var agentNames = map[string]struct{}{
+	"datadog-agent": {}, "agent": {}, "trace-agent": {},
+	"process-agent": {}, "system-probe": {}, "security-agent": {},
+	"dogstatsd": {},
+}
+
+// DiagnosePortSuite displays information about the ports used in the agent configuration
+func DiagnosePortSuite(_ diagnosis.Config, _ sender.DiagnoseSenderManager) []diagnosis.Diagnosis {
```

Comment:
Why are we passing `diagnosis.Config` and `sender.DiagnoseSenderManager` if we do not used them? I think it would be better to not pass them if we do not needed them.